themes/src/main/resources/theme/keycloak-preview/account/messages/messages_en.properties

```diff
@@ -9,6 +9,7 @@ done=Done
 cancel=Cancel
 remove=Remove
 update=Update
+loadingMessage=Keycloak Account Console is Loading ...
```

Comment:
I believe "loading" shouldn't start with capital L.

Reply:
Perhaps do we even need this message? One more thing to productize... IMHO just a spinner animation is clear enough.